src/todo.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-dsibale no-loop-func */
+const addInput = document.querySelector('.todo-input');
+const todos = localStorage.getItem('todos') !== null ? JSON.parse(localStorage.getItem('todos')) : [];
+let index = todos.length;
+const todoDiv = document.querySelector('.todo-list');
+
+const render = () => {
+  if (todos !== null) {
+    todoDiv.innerHTML = '';
+    todos.forEach((todo) => {
+      const check = todo.completed === true ? 'checked' : '';
+      todoDiv.innerHTML += `
+        <div class="todos">
+          <input type="checkbox" class="todo-check" value="${todo.completed}" ${check}>
+          <input type="text" class="todo-desc" value="${todo.desc}">
+          <i class="fa-solid fa-bars"></i>
+          <button type="button" class="trash-btn">delete</buttom>
+        </div>
+      `;
+    });
+  }
+  for (let i = 0; i < todoDiv.querySelectorAll('.todos').length; i += 1) {
+    //  update checked status
+
+    todoDiv.querySelectorAll('.todos')[i].querySelector('.todo-check').addEventListener('click', () => {
+      todos[i].completed = !todos[i].completed;
+      localStorage.setItem('todos', JSON.stringify(todos));
+    });
+
+    //  change color and icon on input focus
+
+    todoDiv.querySelectorAll('.todos')[i].addEventListener('focusin', () => {
+      todoDiv.querySelectorAll('.todos')[i].classList.add('active');
+      todoDiv.querySelectorAll('.todos')[i].querySelector('.todo-desc').classList.add('active');
+      todoDiv.querySelectorAll('.todos')[i].querySelector('.trash-btn').style.display = 'block';
+      todoDiv.querySelectorAll('.todos')[i].querySelector('.fa-bars').style.display = 'none';
+    });
+
+    todoDiv.querySelectorAll('.todos')[i].querySelector('.trash-btn').addEventListener('click', () => {
+      todos.splice(i, 1);
+      for (let a = i; a < todos.length; a += 1) {
+        todos[a].index -= 1;
+      }
+      index -= 1;
+      localStorage.setItem('todos', JSON.stringify(todos));
+      render();
+    });
```

Comment:
- [ ] Good Job for Implementing all the CRUD operations and functionality. But As stated in the project requirement, Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1). So When delete the first task from the List, the  index of the next task(2) not set to 1, Please try to fix it. You can checkout the below image for reference,
The 3 To do List with correct and unique index value :
![Capture](https://user-images.githubusercontent.com/93598189/193421238-35c320be-b390-471d-9706-d11aa8ae33b5.PNG)

The remaining 2 To do list after deleting the first index from the List : 
![Capture1](https://user-images.githubusercontent.com/93598189/193421261-d2058e01-6a9e-4567-a590-6743a27a1cc2.PNG)